### DISCUSSION TITLE
Server Connect Error: Error: websocket error

### DISCUSSION THREAD

**Discussion by fgagliazzo** [jhuckaby/Cronicle#511] [General]
how can i use hostname or url instead of ip?

for example: 
Log Watcher: Connecting to server: http://MY_SERVER:3012...

--

Log Watcher: Connecting to server: http://xxx.xx.xx.35:3012...
Log Watcher: Server Connect Error: Error: websocket error (http://xxx.xx.xx.35:3012)
Log Watcher: Server Connect Error: Error: websocket error (http://xxx.xx.xx.35:3012)

**Comment by jhuckaby** [comment]
https://github.com/jhuckaby/Cronicle#web_socket_use_hostnames

**Reply by fgagliazzo** [reply]
i tried this, but the result is the same.

![image](https://user-images.githubusercontent.com/108342884/183078725-ebd7ea4c-fc94-4ae3-86a7-4464023dab6e.png)

/opt/cronicle/bin/control.sh stop: Cronicle Daemon stopped
/opt/cronicle/bin/control.sh start: Starting up Cronicle Daemon...
/opt/cronicle/bin/control.sh start: Cronicle Daemon started
